src/animation/AnimationClip.js

```diff
@@ -460,6 +463,12 @@ Object.assign( AnimationClip.prototype, {
 
 		return new AnimationClip( this.name, this.duration, tracks, this.blendMode );
 
+	},
+
+	toJSON: function () {
```

Comment:
I've added this method to avoid so the existing code pattern in `Object3D.toJSON()` can be used (which expects `toJSON()` in classes like `Shape`, `Skeleton` and so on).